diff --git a/node_modules/mongoose/index.d.ts b/node_modules/mongoose/index.d.ts
index 799f743..139045b 100644
--- a/node_modules/mongoose/index.d.ts
+++ b/node_modules/mongoose/index.d.ts
@@ -946,6 +946,16 @@ declare module "mongoose" {
     /** Adds static "class" methods to Models compiled from this schema. */
     static(name: string, fn: Function): this;
 
+    methods: {
+      [F in keyof T]: T[F]
+    }
+
+    statics: any;
+
+    
+
+    //Adds statics "class" methods
+
     /** Creates a virtual type with the given name. */
     virtual(name: string, options?: any): VirtualType;
 
